app/controllers/hyrax/downloads_controller.rb

```diff
@@ -105,5 +113,28 @@ def icon_path(asset_id, absolute_path = false)
         end
         ::ThumbnailPathService.icon_path(VisibilityService.visibility_value(file_set.rights_override))
       end
+
+      # Retrieve the content for relation
+      # @param [string] relation
+      # return [binary]
+      def relation_content(relation)
+        case relation
+        when 'preservation_master_file'
```

Comment:
Could `preservation_master_file` be a constant instead of a plain string? Perhaps available via the new `ExtendedContainedFiles`